### ISSUE TITLE
Unresolved directive in <stdin>

### ISSUE THREAD

**Issue by dlehammer** [spring-guides/gs-securing-web#51]
Hi,

The https://spring.io/guides/gs/securing-web/ guide seems to be missing some resources

> Unresolved directive in <stdin>...

![Unresolved directive in stdin 20191022](https://user-images.githubusercontent.com/17439718/67296488-cec33380-f4e8-11e9-8c42-6e01210b8a62.png)

Making the guide hard to follow :)

**Comment by bclozel** [comment]
It's fixed now - the guide itself was alright, it's just a bad rendering from the engine.
Thanks for letting us know!
